src/docs/references/extended-data/index.md

```diff
@@ -59,24 +57,19 @@ collect and store information about users or listings for marketplace
 operators. It can also be used with integrations that want to store e.g.
 a corresponding id in an external service.
 
-### Public metadata
+### Metadata
 
-_Public metadata_ is writable by marketplace operators and can be read
-via all API endpoints returning the corresponding resource. Public
-metadata is supported for users and listings. It can also be used as
+_Metadata_ is writable by marketplace operators and can be read
+via all API endpoints returning the corresponding resource. Metadata
+is supported for users and listings. It can also be used as
 filters when searching via
-[/listings/query](https://www.sharetribe.com/api-reference/marketplace.html#query-listings).
+[/listings/query - in marketplace API](https://www.sharetribe.com/api-reference/marketplace.html#query-listings),
```

Comment:
```suggestion
[/listings/query in Marketplace API](https://www.sharetribe.com/api-reference/marketplace.html#query-listings),
```

same suggestion (remove dash, add capitalization) applies to the other links as wel

Reply:
changed as suggested